cmd/web/config/config_test.go

```diff
@@ -471,3 +489,66 @@ func TestValidatorTypes_AsStringSlice(t *testing.T) {
 		}
 	})
 }
+
+func TestConfig_GetSensored(t *testing.T) {
+
+	const mask = "**masked**"
+
+	cfg := Config{}
+	cfg.Backend.URL = mask
+	cfg.Hash.Key = mask
+	cfg.Server.Profiler.Prefix = mask
+
+	tests := []struct {
+		name string
+		c    Config
+		want Config
+	}{
+		{
+			name: "Testing with valid input",
+			c:    cfg,
+			want: cfg,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ret := tt.c.GetSensored()
+			if ret.Backend != tt.want.Backend {
+				t.Errorf("GetSensored() got = %v, want %v", ret.Backend, tt.want.Backend)
+			}
+			if ret.Hash != tt.want.Hash {
+				t.Errorf("GetSensored() got = %v, want %v", ret.Hash, tt.want.Hash)
+			}
+			if ret.Server.Profiler != tt.want.Server.Profiler {
+				t.Errorf("GetSensored() got = %v, want %v", ret.Server.Profiler, tt.want.Server.Profiler)
+			}
+		})
+	}
+}
+
+func TestString_GetSensored(t *testing.T) {
```

Comment:
With this test you're comparing whether `cfg.String() == cfg.String()`, I think we can safely assume this is indeed true, since none of these functions should change the state of `cfg`.

Perhaps a better test would be to see if the string returned, contains a value you expect, e.g.:
```go
cfg.ListenOn = "hakuna matata"
want = "hakuna matata"
if !strings.Contains(cfg.String(), want) {
// Whelp, 
}
```

Here you test if the phrase from `want` occurs somewhere in the string. Which gives you some guarantee that at least that property is marshalled to a text representation.